modules/aggregation-layers/src/hexagon-layer/hexagon-layer.js

```diff
@@ -96,15 +98,15 @@ export default class HexagonLayer extends AggregationLayer {
       })
     });
 
-    if (oldLayerData !== cpuAggregator.state.layerData) {
-      const {hexagonVertices} = cpuAggregator.state.layerData;
-      this.updateRadiusAngle(hexagonVertices);
-    }
+    const {hexagonVertices} = cpuAggregator.state.layerData;
+    this.updateRadiusAngle(hexagonVertices);
   }
 
   updateRadiusAngle(vertices) {
-    let {radius} = this.props;
     let angle = 90;
+    let radius;
+    const {viewport} = this.context;
+    const {unitsPerMeter} = viewport.getDistanceScales();
 
     if (Array.isArray(vertices)) {
```

Comment:
What is the branch used for? Kepler’s h3 aggregation?

You shouldn’t have to reproject the vertices every time the viewport changes. Radius in common units is a constant.

Reply:
it WAS used when kepler.gl has its own aggregator and aggregate points to h3 hexagons. However, we don't need to use this anymore, since we are not aggregate points into h3. For future development, when we want to create a h3 aggregation layer, we can revisit this

Reply:
A H3 aggregation layer should use the H3HexagonLayer. If this is no longer used by kepler we should mark it as deprecated.

Reply:
Actually `hexagonVertices` is not documented anywhere. Since kepler has never depended on v8 we should just remove it.